homeassistant/components/input_boolean.py

```diff
@@ -140,6 +140,11 @@ def is_on(self):
         return self._state
 
     @asyncio.coroutine
+    def async_restore_state(self, state):
```

Comment:
async restore state should not pass in states but instead pass keyword args with properties. Platforms are not aware how their properties are reresented in the state machine.

```python
# Always add **kwargs so we can catch future new properties
def async_restore_state(self, is_on, **kwargs):
    self._state = is_on
```